pxd.c

```diff
@@ -1776,6 +1778,55 @@ static ssize_t pxd_inprogress_show(struct device *dev,
 	return sprintf(buf, "%d", atomic_read(&pxd_dev->ncount));
 }
 
+static int pxd_nodewipe_cleanup(struct pxd_context *ctx)
+{
+	struct list_head *cur;
+
+	if (ctx->fc.connected) {
+		return -EINVAL;
+	}
+
+	if (ctx->num_devices == 0) {
+		return 0;
+	}
+
+	spin_lock(&ctx->lock);
+	list_for_each(cur, &ctx->list) {
+		struct pxd_device *pxd_dev = container_of(cur, struct pxd_device, node);
+
+		disableFastPath(pxd_dev, true);
+	}
+	spin_unlock(&ctx->lock);
+
+	return 0;
+}
+
+static ssize_t pxd_release_store(struct device *dev,
+			struct device_attribute *attr, const char *buf, size_t count)
+{
+	static const char wipemagic[] = "P0RXR3l3@53";
```

Comment:
Writing a magic code to a control device is basically what an ioctl is. We already have that mechanism, with 10 ioctls implemented. If this interface is insufficient, then it should be extended. If it is incorrect, then a new one should be implemented and everyone migrated.
It's up to you but I don't agree with this roundabout way.